<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimSwitch</class>
 <widget class="QWidget" name="SimSwitch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Leds_sim</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Port address:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAddress">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Output port address, 0-255</string>
   </property>
   <property name="text">
    <string>00</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Vystup">
   <property name="geometry">
    <rect>
     <x>322</x>
     <y>130</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Switch value:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="switchBox_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>95</y>
     <width>35</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="switchBox_6">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>95</y>
     <width>35</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="switchBox_4">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>95</y>
     <width>35</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="switchBox_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>95</y>
     <width>35</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="switchBox_1">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>95</y>
     <width>35</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="switchBox_0">
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>95</y>
     <width>35</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="switchBox_2">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>95</y>
     <width>35</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="switchBox_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>95</y>
     <width>35</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
